net-libs/shairplay/shairplay-0_pre20160304.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools eutils
```

Comment:
eutils is only used for prune_libtool_files, and can be removed once that is changed.